contracts/interfaces/pause/IProtocolPauseAdmin.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.23;
+
+/// @title ProtocolPauseAdmin
+/// @notice Contract that allows the pausing and unpausing of the protocol. It allows adding and removing
+/// pausable contracts, which are contracts that implement the `IPausable` interface.
+/// @dev The contract is restricted to be used only the admin role defined in the `AccessManaged` contract.
+/// NOTE: If a contract is upgraded to remove the `IPausable` interface, it should be removed from the list of pausables
+/// before the upgrade, otherwise pause() and unpause() will revert.
+interface IProtocolPauseAdmin {
```

Comment:
please add a view function to return the current state of the protocol.  Is it paused or not?  Also, return if a contract is paused or not?

Reply:
Each contract that can be paused has a `paused()` method that allows to check for individual state.
The implementation has `function isAllProtocolPaused() external view returns (bool)` but you are right it on not in the interface.